types/p-lazy/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for p-lazy 1.0
+// Project: https://github.com/sindresorhus/p-lazy#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = PLazy;
+
+declare class PLazy<T> extends Promise<T> {
+    static from<T>(fn: () => T | PromiseLike<T>): PLazy<T>;
```

Comment:
Sorry I got confused for a second there with Promise constructor taking two parameters instead of a callback with two optional parameters.